tests/functional/aws-node-sdk/test/object/mpu.js

```diff
@@ -6,31 +6,82 @@ import BucketUtility from '../../lib/utility/bucket-util';
 const bucket = 'object-test-mpu';
 const objectKey = 'toAbort&<>"\'';
 
-const throwErr = (str, err) => {
-    process.stdout.write(`${str}: ${err}\n`);
-    throw err;
-};
-
-const checkValues = (res, uploadId, displayName, cb) => {
-    const prefix = res.Prefix ? res.Prefix : undefined;
-    const delimeter = res.Delimiter ? res.Delimiter : undefined;
-    assert.deepStrictEqual(res.KeyMarker, '');
-    assert.deepStrictEqual(res.UploadIdMarker, '');
-    assert.deepStrictEqual(res.Prefix, prefix);
-    assert.deepStrictEqual(res.Delimiter, delimeter);
-    assert.deepStrictEqual(res.Uploads.length, 1);
-    assert.deepStrictEqual(res.Uploads[0].UploadId, uploadId);
-    assert.deepStrictEqual(res.Uploads[0].StorageClass, 'STANDARD');
-    assert.deepStrictEqual(res.Uploads[0].Owner.DisplayName, displayName);
-    cb();
-};
-
-describe('aws-node-sdk test suite of listMultipartUploads', () => {
+// Get the expected object of listMPU API.
+function getExpectedObj(res, data) {
+    // If `maxUploads` is not given as a parameter, it should default to 1000.
+    const maxUploads = data.maxUploads === undefined ? 1000 : data.maxUploads;
+
+    // If `MaxUploads` is defined as 0, `IsTruncated` is set to `false` despite
+    // the fact that there may be multipart uploads in the bucket.
+    if (maxUploads === 0) {
+        return {
+            Bucket: bucket,
+            KeyMarker: '',
+            UploadIdMarker: '',
+            MaxUploads: 0,
+            IsTruncated: false,
+            Uploads: [],
+            CommonPrefixes: [],
+        };
+    }
+
+    const { prefixVal, delimiter, uploadId, displayName, userId } = data;
+    const initiated = new Date(res.Uploads[0].Initiated.toISOString());
+    const expectedObj = {
+        Bucket: bucket,
+        KeyMarker: '',
+        UploadIdMarker: '',
+        NextKeyMarker: objectKey,
+        Prefix: prefixVal,
+        Delimiter: delimiter,
+        NextUploadIdMarker: uploadId,
+        MaxUploads: maxUploads,
+        IsTruncated: false,
+        Uploads: [{
+            UploadId: uploadId,
+            Key: objectKey,
+            Initiated: initiated,
+            StorageClass: 'STANDARD',
+            Owner:
+            {
+                DisplayName: displayName,
+                ID: userId,
+            },
+            Initiator:
+            {
+                DisplayName: displayName,
+                ID: userId,
+            },
+        }],
+        CommonPrefixes: [],
+    };
+
+    // If no `prefixVal` is given, it should not be included in the response.
+    if (!prefixVal) {
+        assert.deepStrictEqual(res.Prefix, undefined);
```

Comment:
For separation of concerns, please move these assertions to the `checkValues` function. This way `getExpectedObj` does what it says it does, getting/building the expected object.
Tip: Since prefix is only a string `assert.strictEqual` should be used. `deepStrictEqual` is used typically for object assertions.